spice/cfg/tests.cfg

```diff
@@ -44,7 +44,7 @@ include run.cfg
 http_proxy_ip = 192.168.215.1
 http_proxy_ipv6 = fddd:c:c:c::1
 http_proxy_port = 3128
-http_proxy_fqdn = proxy.spice.brq.redhat.com
+http_proxy_fqdn = proxy.spice_br.com
```

Comment:
Maybe add a comment that this hostname comes from /etc/hosts
To easy understand.